docs/storage/lumio/advanced.md

```diff
@@ -1,138 +1,318 @@
-# Advanced usage of LUMI-O  
+# Sharing access to data
```

Comment:
I would use "Sharing data" as the name for this page. It makes menu navigation easier.